### ISSUE TITLE
Improved watering can functionality

### ISSUE THREAD

**Issue by yeahpng** [mim1q/ConvenientDecor#11]
If permanent farmland hydration is disabled, watering cans should still be able to normally hydrate farmland (this would be useful for using the mod along with moisturization). Additionally, you should only be able to place it when sneaking and using it on crops should hydrate the farmland beneath them.

**Comment by mim1q** [comment]
Good point, should be pretty simple to implement! I'll comment here again once I do that :grin:
